include/tig/main.h

```diff
@@ -19,7 +19,7 @@
 #include "tig/util.h"
 
 struct commit {
-	char id[SIZEOF_REV];		/* SHA1 ID. */
+	char id[SIZEOF_REV];		/* Hash ID. */
```

Comment:
this looks good apart from one comment about the new value.
I haven't used sha256, I'll try it if I get around to.
It's a shame we can't reuse existing unit tests without modification.

Reply:
Actually `git fast-export` and `git fast-import` allow to port the tests without too much effort and also to get a nice sandbox from an existing repository.